composer.json

```diff
@@ -9,7 +9,7 @@
   },
   "require": {
     "ext-json": "*",
-    "twig/twig": "^1.41.1",
+    "twig/twig": "^1.41.1 || ^2.12",
```

Comment:
Let’s make this just v2. Since this isn’t installed as a composer dependency, it won’t be able to see what version the main project uses.

Reply:
Done!

Reply:
Nope, I meant it should simply be `^2.12`

Reply:
My bad. Fixed.